### PR TITLE
cleanup: avoid double import of kubernetes library in e2e utils

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: e2e/utils.go [hunk 1]
```diff
@@ -22,7 +22,6 @@ import (
 	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/util/wait"
 	utilyaml "k8s.io/apimachinery/pkg/util/yaml"
-	"k8s.io/client-go/kubernetes"
 	clientset "k8s.io/client-go/kubernetes"
```
Comment (nixpanic) [inline e2e/utils.go:25]: please run compile tests locally. I expect this to fail a simple `make e2e.test`

File: e2e/utils.go [hunk 2]
```diff
@@ -23,12 +21,15 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	utilyaml "k8s.io/apimachinery/pkg/util/yaml"
 	"k8s.io/client-go/kubernetes"
-	clientset "k8s.io/client-go/kubernetes"
+
+	//clientset "k8s.io/client-go/kubernetes"
```
Comment (nixpanic) [inline e2e/utils.go:25]: don't comment it out, just remove it completely

Comment (humblec) [reply e2e/utils.go:25]: Its just that I didnt notice that its commented out :)..

**Comment by humblec** [comment]
@Madhu-1 @nixpanic ptal.

File: e2e/utils.go [hunk 3]
```diff
@@ -23,12 +21,14 @@ import (
...
 	"k8s.io/client-go/kubernetes"
-	clientset "k8s.io/client-go/kubernetes"
+
 	"k8s.io/kubernetes/pkg/client/conditions"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
 	e2epv "k8s.io/kubernetes/test/e2e/framework/pv"
 	testutils "k8s.io/kubernetes/test/utils"
+
+	"github.com/ceph/ceph-csi/internal/util"
```
Comment (Madhu-1) [inline e2e/utils.go:31]: cephcsi packages should be imported second followed by third-party packages

**Comment by humblec** [comment]
@Madhu-1 @nixpanic ptal.

**Comment by humblec** [comment]
@nixpanic ptal on latest version.

**Pull request merged**